coderd/provisionerdserver/provisionerdserver.go

```diff
@@ -1027,6 +1035,7 @@ func (s *server) CompleteJob(ctx context.Context, completed *proto.CompletedJob)
 		}
 
 		var completedError sql.NullString
+		var completedErrorCode sql.NullString
```

Comment:
This doesn't seem to be ever assigned, it's use could be confusing for future change(r)s.

Reply:
Removing it.